multiversx_sdk/core/transactions_outcome_parsers/resources.py

```diff
@@ -38,4 +38,37 @@ def __init__(self,
                  transaction_results: List[SmartContractResult],
                  transaction_logs: TransactionLogs) -> None:
         self.transaction_results = transaction_results
```

Comment:
in specs, there is a reference to `direct_smart_contract_call_outcome: SmartContractCallOutcome;` should this be set here also?

Reply:
That was added after this implementation was done. It is not needed now, will be added in a future PR.